src/hint_processor/builtin_hint_processor/skip_next_instruction.rs

```diff
@@ -0,0 +1,11 @@
+use crate::vm::{errors::vm_errors::VirtualMachineError, vm_core::VirtualMachine};
+
+/*
+This hint doesn't belong to the Cairo common library
+It's only added for testing proposes
+*/
+
+pub fn skip_next_instruction(vm: &mut VirtualMachine) -> Result<(), VirtualMachineError> {
```

Comment:
This could be `pub(crate)`.

Reply:
i'm thinking this could be exposed for maximum composability with other test-oriented hints